features/application/MainNavigation.feature

```diff
@@ -116,11 +64,7 @@ Feature: index
     Examples:
       | Selector                                     | Title              |
       | $main-nav .nav-item-home a                   | Overview           |
-      | $main-nav .nav-item-zone-list-view a         | Zones              |
+      | $main-nav .nav-item-zone-list-view a         | Zone CPs           |
```

Comment:
I kinda guessed this should say `Zone CPs` but lemme know if not and I can put back

Reply:
Hm, probably, yes. In any case, we might not have these sidebar entries for long anyway.